docs/source/en/_toctree.yml

```diff
@@ -25,147 +25,146 @@
     title: Agents
   title: Tutorials
 - sections:
-  - sections:
-      - local: tasks/sequence_classification
-        title: Text classification
-      - local: tasks/token_classification
-        title: Token classification
-      - local: tasks/question_answering
-        title: Question answering
-      - local: tasks/language_modeling
-        title: Causal language modeling
-      - local: tasks/masked_language_modeling
-        title: Masked language modeling
-      - local: tasks/translation
-        title: Translation
-      - local: tasks/summarization
-        title: Summarization
-      - local: tasks/multiple_choice
-        title: Multiple choice
```

Comment:
Not sure what happened here.. 

My ruff version is 0.0.259, black version is 23.1.0

Reply:
Neither ruff or black will touch a yaml file. Nor sure what happened either but it needs to be reverted.